django_usaid_clinicfinder/clinicfinder/tasks.py

```diff
@@ -151,18 +151,25 @@ def run(self, lookuppointofinterest_id, **kwargs):
             if response["type"] == "SMS" and "sent" not in response:
                 # send via Vumi
                 sender = self.vumi_client()
-                content = response["template"].replace(
-                    "{{ results }}", response["results"])
-                if len(content) <= settings.LOCATION_RESPONSE_MAX_LENGTH:
-                    # Defaults to 320
+                if response["results"] != "":
+                    content = response["template"].replace(
+                        "{{ results }}", response["results"])
+                    vumiresponse = False
+                    if len(content) <= settings.LOCATION_RESPONSE_MAX_LENGTH:
+                        # Defaults to 320
+                        vumiresponse = sender.send_text(
+                            response["to_addr"], content)
+                        lookuppoi.response["sent"] = "true"
+                        l.info("Sent message to <%s>" % response["to_addr"])
+                    else:
+                        l.info(
+                            "Message not sent to <%s>. Too long at <%s> chars." %
+                            (response["to_addr"], str(len(content))))
```

Comment:
Not sure if `lookuppoi.response["sent"]` defaults to `"false"` or should be set here.